### PR TITLE
Hail2 overview

### PR DESCRIPTION
**Pull request by maccum**

### CODE REVIEW

File: python/hail/docs/hail2_tutorials/hail2-overview.ipynb [hunk 1]
```diff
@@ -0,0 +1,2215 @@
...
...VGFVpxyPHRxiltP\nVxRRJdWM0ZJHI06plULIIhq9FM3AE6FeCRgt+3zD4VFuPzSC4wjPnm+imnFsvMotB2s0uwkPv7jA\nXCPixoM+//DWaW6erq9mhP0M0nGE6XqJ6olxWr2EF+db3PWqg5wvGt1LvkM3zkizjFcdGqESvHxn\n2m8zCgaqfKIko9VLGKsEV3QXO1hfX/JdDo2WON/okmWKs8m78mFl+nY3btZjwcRc0touvF87u8xS\nOyZOlYMjJW49UMN1hWfPNzi73CVKUgLPJVV4ablD1fcIXYeK5zLXioiShOVWTDl0aXYTICPL8h5W\njivUfZdWL6FW8khVmB6pMp4kvNTo0VVl5sQ4x8crPH2uSTdOiVIl9B2OjVe4YarCeDlgohowUQu4\n7UCtaCzOu7yOlPJ9L863OTha4pbp2kXdWtdmkGOVYLXBWVF6cUY3yfBdoVYKqJe8VzQ8b2dX0bXV\nSY4IE5Vg06URG1BndpoFE7OutV14//6lBo+eWmay6qEIj59Z5tnzDSbrIY12QrXkk6ny92eXWekm\nRBmMBi4esNKOCVxhoZXQjVIcF9JUyVTxvbw3UjX0CXyHVpxybLKG7wo3TFcouw7HOhHPX+jwrTdN\ncGy8yve+9iidOOXMQoeFVsSJqSq+Ixwdr1AveTS7CV+bXaaXZIyWAybrAaHnstCKCXyHahFgNjPI\nrF9CqJd8qiH4rkOcZIhc3PC8nV1Ft9q4awPqzE6zYGLWNd/sFeMq2jw+u8xTLzXycRHtHjdN1hir\n+rw43+brcw1ef9NkPj/R+SaeKyy1Y84tdzg11+LIRIk0VRq9vLrr8FiIKjR6+UC/Q2N1ojjvtjtd\nCzk2XuEbD48wWQ85Nd9ivhlxZLTCm77xIEfGq2SqLLYiQDg2UeE1J8bzHlRF9VI3TplvRYxVA+Ya\nXZq9iG6cMD0SstyOOFgPGK++PC3I5UoO/RJCNfC40OwRJxmZZkzWShc1PG93V9GtVCfZgDqz0yyY\nmHWtdBPmVjrMNSKW2jFpltFLUgSH5U5MOc270gauh6NwbrmH7zl4IjS6MYHvEQYuF1Z6tOOEkudR\nK3l84+FRVjr5/kwzJqp5cJmqBZQDj1cfG+M1RXvFoZHyK+Zo6k//UfE9PCfPGLtRQi9xGK/4QN6L\nKfCcovdWPvZid
```
Comment (patrick-schultz) [inline python/hail/docs/hail2_tutorials/hail2-overview.ipynb]: We shouldn't call this `kt` anymore. No sure what to rename it to.

Comment (patrick-schultz) [inline python/hail/docs/hail2_tutorials/hail2-overview.ipynb]: `kt.show()` works.

Comment (patrick-schultz) [inline python/hail/docs/hail2_tutorials/hail2-overview.ipynb]: Likewise.

**Comment by patrick-schultz** [comment]
@tpoterba Didn't see you reassigned this.

**Comment by tpoterba** [comment]
You had a bunch of other open ones, and I was planning to take a look anyway. You can have it back if you want!

**Comment by patrick-schultz** [comment]
I'll let you take it from here, I got all my thoughts in.

**Pull request closed without merging**